django/dispatch/dispatcher.py

```diff
@@ -208,6 +209,8 @@ def send_robust(self, sender, **named):
             try:
                 response = receiver(signal=self, sender=sender, **named)
             except Exception as err:
+                logger = logging.getLogger("django.dispatch.dispatcher")
```

Comment:
This call should be done once at the module level, see e.g. `django/core/handlers/asgi.py`

Reply:
ok i will make this change